core/ui/ViewToolbar/copy-tiddler.tid

```diff
@@ -0,0 +1,23 @@
+title: $:/core/ui/Buttons/copy-tiddler
+tags: $:/tags/ViewToolbar
+caption: {{$:/core/images/copy-clipboard}} {{$:/language/Buttons/CopyTiddler/Caption}}
+description: {{$:/language/Buttons/CopyTiddler/Hint}}
+
+\define copyTiddlerButton()
+\whitespace trim
+<$button tooltip={{$:/language/Buttons/CopyTiddler/Hint}} aria-label={{$:/language/Buttons/CopyTiddler/Caption}} class=<<tv-config-toolbar-class>>>
+<$action-clipboard $tiddler=<<currentTiddler>> $text="hello"/>
```

Comment:
actions should never be activated inside the button body. It causes refresh problems. 
Best practice is: `<$button actions=<<your-action-procedure>>...>` instead.